test/extensions/quic_listeners/quiche/integration/BUILD

```diff
@@ -32,9 +32,10 @@ envoy_cc_test(
 
 envoy_cc_test(
     name = "quic_http_integration_test",
-    size = "medium",
+    size = "large",
     srcs = ["quic_http_integration_test.cc"],
     data = ["//test/config/integration/certs"],
+    shard_count = 12,
```

Comment:
protocol_integration_test has only 5 shards - are 12 necessary?  runtime of 50-60s is pretty normal so as long as it doesn't take several minutes I think we're good.

Reply:
This tests runs a combination of {IPv4, IPv6} and {gQUIC with Quic Crypto, gQUIC with TLS, iQUIC with TLS}. So I reduced it to 6.